src/Orderings.jl

```diff
@@ -0,0 +1,29 @@
+import Base.Order: lt, Ordering
+export LenLexOrder
+
+"""
+    struct LenLexOrder{T<:Alphabet} <: Ordering
+
+Basic structure representing Length+Lexicographic (left-to-right) ordering of
+the words over given Alphabet. Lexicographing ordering of an Alphabet is
+implicitly specified inside Alphabet struct.
+"""
+struct LenLexOrder{T<:Alphabet} <: Ordering
```

Comment:
maybe we don't even need to include `Order` in the name? what do you think?

Reply:
For this kind of questions I can just wave with 🇨🇭 flag and say I remain neutral. De gustibus non est disputandum.